sql/2020/18_Page_Weight/18_01b.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 18_01b: Distribution of page weight by resource type and client (2019).
+SELECT
+  percentile,
+  _TABLE_SUFFIX as client,
+  APPROX_QUANTILES(ROUND(bytesTotal / 1024, 2), 1000)[OFFSET(percentile * 10)] AS total_kbytes,
+  APPROX_QUANTILES(ROUND(bytesHtml / 1024, 2), 1000)[OFFSET(percentile * 10)] AS html_kbytes,
+  APPROX_QUANTILES(ROUND(bytesJS / 1024, 2), 1000)[OFFSET(percentile * 10)] AS js_kbytes,
+  APPROX_QUANTILES(ROUND(bytesCSS / 1024, 2), 1000)[OFFSET(percentile * 10)] AS css_kbytes,
+  APPROX_QUANTILES(ROUND(bytesImg / 1024, 2), 1000)[OFFSET(percentile * 10)] AS img_kbytes,
+  APPROX_QUANTILES(ROUND(bytesOther / 1024, 2), 1000)[OFFSET(percentile * 10)] AS other_kbytes,
+  APPROX_QUANTILES(ROUND(bytesHtmlDoc / 1024, 2), 1000)[OFFSET(percentile * 10)] AS html_doc_kbytes
+FROM
+  `httparchive.summary_pages.2019_08_01_*`,
```

Comment:
The 2019 Web Almanac used the July dataset, so let's reuse that for consistency.
```suggestion
  `httparchive.summary_pages.2019_07_01_*`,
```